docs/DeveloperGuide.md

```diff
@@ -630,9 +598,33 @@ otherwise)
 
       Use Case ends.
 
+* 1b. Logged-in Owner specifies Password field and not Confirm Password field.
+
+    * 1b1. US requests Logged-in Owner to specify both Password and Confirm Password field.
+
+      Use Case ends.
+
+* 1c. Logged-in Owner specifies Confirm Password field and not Password field.
+
+    * 1c1. US requests Logged-in Owner to specify both Password and Confirm Password field.
```

Comment:
What about password mismatch, when "password" is not equal to "confirm password"?

Reply:
Thanks for catching! Added as extension 1d